docker/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '2.3'
+
+services:
+
+  web:
+    build:
+      context: ..
+      dockerfile: docker/nginx/Dockerfile
+    ports:
+      - 80:80
```

Comment:
Using a lesser used port will prevent `make install` issues when another service is already bound to port 80.

Reply:
Good suggestion, I made this configurable. You need to run `make set-env` again, and then you can specify the ports of mysql and php yourself in `.env`.